-------------------------------------------------------------------------------
Test set: com.lmx.project.service.UserServiceTest
-------------------------------------------------------------------------------
Tests run: 5, Failures: 1, Errors: 4, Skipped: 0, Time elapsed: 18.851 s <<< FAILURE! - in com.lmx.project.service.UserServiceTest
testAddUser  Time elapsed: 1.909 s  <<< ERROR!
org.mybatis.spring.MyBatisSystemException: 
nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.insert
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testAddUser(UserServiceTest.java:24)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 

### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.insert
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testAddUser(UserServiceTest.java:24)
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testAddUser(UserServiceTest.java:24)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testAddUser(UserServiceTest.java:24)

userRegister  Time elapsed: 0.009 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <-1> but was: <0>
	at com.lmx.project.service.UserServiceTest.userRegister(UserServiceTest.java:55)

testUpdateUser  Time elapsed: 1.102 s  <<< ERROR!
org.mybatis.spring.MyBatisSystemException: 
nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.updateById
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testUpdateUser(UserServiceTest.java:32)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 

### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.updateById
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testUpdateUser(UserServiceTest.java:32)
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testUpdateUser(UserServiceTest.java:32)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testUpdateUser(UserServiceTest.java:32)

testGetUser  Time elapsed: 1.086 s  <<< ERROR!
org.mybatis.spring.MyBatisSystemException: 
nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.selectById
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testGetUser(UserServiceTest.java:44)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 

### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.selectById
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testGetUser(UserServiceTest.java:44)
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testGetUser(UserServiceTest.java:44)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testGetUser(UserServiceTest.java:44)

testDeleteUser  Time elapsed: 1.015 s  <<< ERROR!
org.mybatis.spring.MyBatisSystemException: 
nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.deleteById
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testDeleteUser(UserServiceTest.java:38)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 

### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
### The error may exist in com/lmx/project/mapper/UserMapper.java (best guess)
### The error may involve com.lmx.project.mapper.UserMapper.deleteById
### The error occurred while executing an update
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testDeleteUser(UserServiceTest.java:38)
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testDeleteUser(UserServiceTest.java:38)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.lmx.project.service.UserServiceTest.testDeleteUser(UserServiceTest.java:38)

